src/OSPSuite.Core/Domain/Services/ObserverBuilderTask.cs

```diff
@@ -68,7 +68,7 @@ public void CreateObservers(IBuildConfiguration buildConfiguration, IModel model
       private IEnumerable<IMoleculeBuilder> moleculeBuildersValidFor(MoleculeList moleculeList, IEnumerable<IMoleculeBuilder> allMolecules)
       {
          if (moleculeList.ForAll)
-            return allMolecules.Where(molecule => !moleculeList.MoleculeNamesToExclude.Contains(molecule.Name));
+            return allMolecules.Where(molecule => molecule.IsFloating && !moleculeList.MoleculeNamesToExclude.Contains(molecule.Name));
```

Comment:
@Yuri05 Hard for me to tell
But it feels like ForAll does not apply for non floating molecule. IF you want CYP etc,,, I am cool in saying they should be named excplitely

This is definitely how it was intended at the beginning. Wonder why we never saw it before

Reply:
Hard to say for me if we should exclude non-floating molecules here...